server/app/services/item_tag.py

```diff
@@ -0,0 +1,19 @@
+from typing import List
+
+from ..models.tag import Tag
+from ..models.user import User
+from ..models.item_tag import ItemTag
+from ..schemas.tag import TagCreate
+from ..database import get_session
+
+
+def create_item_items_tags(item_id: int, tags: List[TagCreate], user: User):
+    tags = [TagCreate(name=user.username)] if not tags else tags
```

Comment:
The line tags = [TagCreate(name=user.username)] if not tags else tags could potentially overwrite the input parameter which is generally not a good practice. A separate variable should be used if modifying the list of tags is necessary.

Reply:
Thanks! Added a fix